team/team.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+   </title>
+  </head>
+  <body>
+    <h2>Epic Pi Bots team members</h2>
+    <hr>
+    <h1>Owner and main creator of website </h1>
```

Comment:
*laughter fades to depression*
```suggestion
    <h2>Owner and main creator of website </h2>
```